sql/core/src/test/resources/sql-tests/inputs/interval.sql

```diff
@@ -170,6 +170,21 @@ select
   null - interval '2' year,
   null + interval '2' hour,
   null - interval '2' hour;
+select
```

Comment:
there is a section at the beginning of this file `-- multiply and divide an interval by a number`. Can we put the null tests there?

Reply:
Done